sentry/src/main/java/io/sentry/protocol/Feedback.java

```diff
@@ -0,0 +1,233 @@
+package io.sentry.protocol;
+
+import io.sentry.ILogger;
+import io.sentry.JsonDeserializer;
+import io.sentry.JsonSerializable;
+import io.sentry.JsonUnknown;
+import io.sentry.ObjectReader;
+import io.sentry.ObjectWriter;
+import io.sentry.SentryLevel;
+import io.sentry.util.CollectionUtils;
+import io.sentry.vendor.gson.stream.JsonToken;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+// Specs can be found at https://develop.sentry.dev/sdk/data-model/envelope-items/#user-feedback
+
+public final class Feedback implements JsonUnknown, JsonSerializable {
+  public static final String TYPE = "feedback";
+
+  private @NotNull String message;
+  private @Nullable String contactEmail;
+  private @Nullable String name;
+  private @Nullable SentryId associatedEventId;
+  private @Nullable SentryId replayId;
+  private @Nullable String url;
```

Comment:
I guess we have to set these two from the scope (url can be scope.screen), but I guess you'll take care of this later in SentryClient

Reply:
I forgot to add it 😄 
I added it directly in `ISentryClient`